.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,14 +1,17 @@
-blank_issues_enabled: true
+blank_issues_enabled: false
```

Comment:
Ah, I was wondering how people were creating issues without a template 😄

Reply:
I did this because I previously moved the feature requests, and wanted maintainers to be allowed to create any issue. But now I have added the task then we can remove this again. GH needs more RBAC.